packages/canvas/canvas-renderer/src/CanvasRenderer.ts

```diff
@@ -278,9 +278,20 @@ export class CanvasRenderer extends AbstractRenderer
                     context.fillStyle = this._backgroundColorString;
                     context.fillRect(0, 0, this.width, this.height);
                 }
-            } // else {
-            // TODO: implement background for CanvasRenderTarget or RenderTexture?
-            // }
+            }
+            else
+            {
+                renderTexture = (renderTexture as BaseRenderTexture);
+                renderTexture._canvasRenderTarget.clear();
+
+                const clearColor = renderTexture.clearColor;
+
+                if (clearColor[0] !== 0 || clearColor[1] !== 0 || clearColor[2] !== 0 || clearColor[3] !== 0)
```

Comment:
Don't we only need to check for `clearColor[3] > 0`? We should probably support `[0, 0, 0, 1]` as a clearColor (black).

Reply:
What do you mean "support as a clearColor"? If `clearColor[3] > 0` (true for [0, 0, 0, 1] black), then will be cleared inside the if-block.

Reply:
I was suggesting just to simplify the logic. We don't need to check for rgb values, only a to know whether to use clearColor or not.